HiP-DataStore/Controllers/ExhibitsController.cs

```diff
@@ -99,7 +108,7 @@ public IActionResult GetById(int id, DateTimeOffset? timestamp = null)
             if (!ModelState.IsValid)
                 return BadRequest(ModelState);
 
-            var status = _entityIndex.Status(ResourceType.Exhibit, id) ?? ContentStatus.Published;
+            var status = _entityIndex.Status(ResourceType.Exhibit, id) ?? ContentStatus.Deleted;
```

Comment:
I think defaulting to `ContentStatus.Deleted` is not perfect here: If you try to get an exhibit that doesn't exist, you now always get `403 Forbidden` instead of `404 Not Found`. I find this a bit misleading. Defaulting to `ContentStatus.Published` seems more reasonable, why did you change it?

(this probably applies to pages/routes/etc. as well, I didn't check that)